addons/cargo/stringtable.xml

```diff
@@ -299,6 +302,7 @@
         </Key>
         <Key ID="STR_ACE_Cargo_size_edenDesc">
             <English>The cargo space required to hold this object (-1 for unloadable)</English>
+            <German>Frachtgröße zum Einladen dieses Objektes (-1 nicht einladbar)</German>
```

Comment:
the `required` part is missing there.
Maybe..
`Benötigter Frachtraum zum...`?

Reply:
"Benötigter Frachtraum um dieses Objekt einzuladen (-1 nicht einladbar)"

*clever clogs* 🥇